src/main/groovy/com/morpheusdata/upcloud/UpcloudProvisionProvider.groovy

```diff
@@ -99,7 +103,10 @@ class UpcloudProvisionProvider extends AbstractProvisionProvider implements Work
 	 */
 	@Override
 	Collection<StorageVolumeType> getRootVolumeStorageTypes() {
-		Collection<StorageVolumeType> volumeTypes = []
+		Collection<StorageVolumeType> volumeTypes = [
+			new StorageVolumeType(code:'upcloudVolume', displayName:'UpCloud MaxIOPS', name:'MaxIOPS', description:'UpCloud MaxIOPS', volumeType:'disk', enabled:true,
+				displayOrder:1, customLabel:true, customSize:true, defaultType:true, autoDelete:true, minStorage:(10L * oneGB), allowSearch:true, volumeCategory:'disk')
```

Comment:
I don't see a `oneGB` variable defined. The compute utility has some predefined constants for this: `ComputeUtility.ONE_GIGABYTE`